src/nl/surf/eduhub_rio_mapper/processing.clj

```diff
@@ -106,13 +106,14 @@
       (relation-handler/after-upsert eduspec job handlers))
     request))
 
-(defn- wrap-phase [[phase f]]
+(defn- wrap-phase [[func-phase f]]
```

Comment:
"func"?  What's that?

Reply:
Renamed to distinguish it from phase in ex-data, but then realized we don't use phase in ex-data anymore. 

Fixed now.